packages/components/inputs/localized-rich-text-input/src/localized-rich-text-input.visualspec.js

```diff
@@ -100,7 +100,7 @@ describe('LocalizedRichTextInput', () => {
     await input.type('Hello world');
 
     // check that there is now a strong tag in the document.
-    numOfTags = await getNumberOfTags('strong');
```

Comment:
Previously there was no actual check here.
It was needed the _focus_ mentioned before to actually test the expected behaviour.